scripts/globals/survival_guide_map.lua

```diff
@@ -0,0 +1,1181 @@
+survivalGuides = {
+    [6] = {
+        value = "West Ronfaure",
+        zoneId = 100,
+        group = 1,
+        groupMask = 1,
```

Comment:
Isn't groupMask always going to be 2 ^ (groupIndex - 1)? Do we need to define all these powers of 2 in our table?